test/libsolidity/semanticTests/abiEncoderV2/calldata_array_short.sol

```diff
@@ -0,0 +1,10 @@
+contract C {
+	function f(uint[] calldata) public {}
+}
+// ====
+// revertStrings: debug
+// compileViaYul: also
+// ----
+// f(uint256[]): 0x20, 0 ->
+// f(uint256[]): 0x20, 1 -> FAILURE, hex"08c379a0", 0x20, 0x2b, "ABI decoding: invalid calldata a", "rray stride"
+// f(uint256[]): 0x20, 2 -> FAILURE, hex"08c379a0", 0x20, 0x2b, "ABI decoding: invalid calldata a", "rray stride"
```

Comment:
Not sure if we should change all those revert reasons to something better while we're at it :-)... but it may unnecessarily confuse tooling that expects them like this...